From dda35189e97fb97cbc0cdf00dbd4b1dda1d2b840 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Mon, 18 Jul 2016 17:13:05 +0000
Subject: [PATCH] OS-5516 vmxnet3s declares wrong sdu Reviewed by: Patrick
 Mooney <patrick.mooney@joyent.com> Approved by: Bryan Cantrill
 <bryan@joyent.com>

---
 usr/src/uts/intel/io/vmxnet3s/vmxnet3_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/uts/intel/io/vmxnet3s/vmxnet3_main.c b/usr/src/uts/intel/io/vmxnet3s/vmxnet3_main.c
index 3532c5c458..2b54baf799 100644
--- a/usr/src/uts/intel/io/vmxnet3s/vmxnet3_main.c
+++ b/usr/src/uts/intel/io/vmxnet3s/vmxnet3_main.c
@@ -1456,7 +1456,7 @@ vmxnet3_attach(dev_info_t *dip, ddi_attach_cmd_t cmd)
 	macr->m_src_addr = dp->macaddr;
 	macr->m_dst_addr = NULL;
 	macr->m_callbacks = &vmxnet3_mac_callbacks;
-	macr->m_min_sdu = VMXNET3_MIN_MTU;
+	macr->m_min_sdu = 0;
 	macr->m_max_sdu = ETHERMTU;
 	macr->m_margin = VLAN_TAGSZ;
 	macr->m_pdata = NULL;
-- 
2.21.0

